### ISSUE TITLE
ForgotPassword POST action has invalid model

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#83]
### Brief bug description

Upon uploading the form of the ForgotPassword action method, `ModelState.IsValid` always fails due to `EmailViewModel.Email` being `null` despite the filled form.

### Repro steps

1. Place a breakpoint into ForgotPassword (POST overload)
2. Navigate to `/en-us/identity/Account/ForgotPassword`
3. Fill out the form
4. Observe the `null` value in debugger

### Expected behavior

The `Email` property has the value from the form.

### Test environment

 - Platform/OS: [e.g. .NET Core 3.1, iOS, Windows]
 - Browser [e.g. Chrome, Safari, Edge Chromium, Edge]
 - Version [e.g. 81]

### Additional context

Add any other context about the problem here. If you found an issue where a similar bug has been reported, add a link here.

### Screenshots

Add links to screenshots, if you deem it appropriate.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
